pynestml/utils/ast_utils.py

```diff
@@ -1316,17 +1318,114 @@ def get_input_port_by_name(cls, input_blocks: List[ASTInputBlock], port_name: st
         return None
 
     @classmethod
-    def get_parameter_by_name(cls, parameters_block: ASTBlockWithVariables, var_name: str) -> ASTDeclaration:
+    def get_parameter_by_name(cls, node: ASTNeuronOrSynapse, var_name: str) -> ASTDeclaration:
         """
         Get the declaration based on the name of the parameter
         :param parameters_block: the parameter block
         :param var_name: variable name to be searched
         :return: declaration containing the variable
         """
-        for decl in parameters_block.get_declarations():
-            for var in decl.get_variables():
-                if var.get_name() == var_name:
-                    return decl
+        for param_block in node.get_parameters_blocks():
+            for decl in param_block.get_declarations():
+                for var in decl.get_variables():
+                    if var.get_name() == var_name:
+                        return decl
+        return None
+
+    @classmethod
+    def get_parameter_variable_by_name(cls, node: ASTNeuronOrSynapse, var_name: str) -> ASTVariable:
+        """
+        Get the declaration based on the name of the parameter
+        :param parameters_block: the parameter block
+        :param var_name: variable name to be searched
+        :return: declaration containing the variable
+        """
+        for param_block in node.get_parameters_blocks():
+            for decl in param_block.get_declarations():
+                for var in decl.get_variables():
+                    if var.get_name() == var_name:
+                        return var
+        return None
+
+    @classmethod
+    def get_internal_by_name(cls, node: ASTNeuronOrSynapse, var_name: str) -> ASTDeclaration:
+        """
+        Get the declaration based on the name of the parameter
```

Comment:
Could you change the docstrings of the `get_` functions in this file to match the function definition?